resources/osx/scripts/postinstall

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+sudo mkdir -p /usr/local/bin
+sudo ln -sf /usr/local/lib/heroku/bin/heroku /usr/local/bin/heroku
+rm -rf ~/.local/share/heroku/cli
```

Comment:
@dickeyxxx some brief googling determined that it is run as root so I would remove this